docs/general/known_issues.rst

```diff
@@ -1,16 +1,14 @@
 Limitations and known issues
 ============================
 
-.. include:: ../includes/top-warning.rst
-
 Reporting issues
 ----------------
 
-For the duration of the pilot, bugs and other issues found in
-*SecureDrop Workstation* should be reported by pilot participants via the
-`support portal <https://support.freedom.press>`_. Issues that are not instance-
-specific will be added to the appropriate public issue tracker by the developers,
-for example:
+Please report sensitive issues that are specific to your instance
+via the `support portal <https://support.freedom.press>`_. 
+
+Bugs and other issues that are not specific to your instance can be reported
+via Github using the following links:
```

Comment:
nit:

```suggestion
via GitHub using the following links:
```

Maybe also "If you encounter a security-related issue, please see [SECURITY.md](https://github.com/freedomofpress/securedrop-workstation/blob/main/SECURITY.md) for instructions on how to privately report it"??

Reply:
Great suggestion! Added :)